source/includes/fact-favorite-query.rst

```diff
@@ -0,0 +1,26 @@
+You can favorite a query so that you can find it later. If you
+load a favorited query, you can change it without
+changing the original.
+
+To add a query to your favorites:
+
+1. Click the :guilabel:`History` button at the top of the pipeline
```

Comment:
Maybe try the stepped procedure RST here as well?

Reply:
Updated.